api_tests/nodes/views/test_node_alternative_citations.py

```diff
@@ -0,0 +1,1098 @@
+from nose.tools import *  # flake8: noqa
+
+from website.util import permissions
+
+from api.base.settings.defaults import API_BASE
+
+from tests.base import ApiTestCase
+
+from tests.factories import (
+    ProjectFactory,
+    RegistrationFactory,
+    AuthUserFactory,
+    AlternativeCitationFactory
+)
+
+class TestCreateAlternativeCitations(ApiTestCase):
```

Comment:
I noticed these tests test combinations of {public, private} x {admin, non-admin contributor, non-contributor} Can you also write tests cases for logged out users?